### PR TITLE
Run samples/ as part of solution tests

### PR DESCRIPTION
**Pull request by asvrada**

### CODE REVIEW

File: samples/TEE_aware/dotnet/README.md [hunk 1]
```diff
@@ -81,8 +81,7 @@ To learn more about writing TEE aware applications, please read the [TEE aware a
 
 ## Running the sample
 
-To run the sample in package mode, use `make run` to launch both the server and the local client that will
-send images located inside `test_samples` for inferencing.
```
Comment (radhikaj) [inline samples/TEE_aware/dotnet/README.md:85]: This line is correct. It says that the images inside the `test_samples` directory will be sent for inferencing.

**Review by radhikaj — APPROVED** [review]
Thanks for the improvements!

File: samples/TEE_aware/dotnet/README.md (hunk 1 above)
Comment (asvrada) [reply samples/TEE_aware/dotnet/README.md:85]: I think this dotnet sample doesn't have this functionality?

**Comment by asvrada** [comment]
Merge, ignoring the unrelated test failure in the pipeline

**Pull request merged**